### ISSUE TITLE
Avoid git push --force

### ISSUE THREAD

**Issue by dankohn** [cncf/landscapeapp#62]
Instead of the current [code](https://github.com/cncf/landscapeapp/blob/master/full_build.sh#L21-L22):
```sh
git push github HEAD:master
git push github HEAD:master --tags --force
```
Can we do:
```sh
git push github HEAD:master --follow-tags
```
See https://stackoverflow.com/a/3745250/1935918

Or do we need the `--force` for some reason?

**Comment by ZeusTheTrueGod** [comment]
It may happen that we already have that tag in a github, thus git push github HEAD:master --follow-tags would return an error with all the further builds.
The idea here is that we want to push HEAD in a normal way, and if it succeeds, then to force a tag on a build.

During my tests I've got this at least once, I am not sure how exactly I've got this problem though. Probably the tag was pushed but then I failed to publish for an npm, that could happen because of the network issue, for example, and then all further builds will fail till we find out what is wrong and remove the tag.

We can completely ignore tags on a github, it is just a good practice to ensure that every npm build is linked to the github.
